src/style.css

```diff
@@ -0,0 +1,95 @@
+* {
+  box-sizing: border-box;
+}
+
+@font-face {
+  font-family: "Avenir-Book";
+  src: url(./fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir-Heavy";
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 700;
+}
+
+.search1 {
```

Comment:
Why is it `search1`? It may be just `search` isn't it?

Reply:
due to a merge error, I thought maybe if something changes, it will pass without errors, but still an error when merge develop into master